.github/workflows/build_documentation.yml

```diff
@@ -46,7 +47,9 @@ jobs:
           sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev
 
           pip install git+https://github.com/huggingface/doc-builder
-          pip install git+https://github.com/huggingface/transformers#egg=transformers[dev]
+          cd transformers
+          pip install .
+          cd ..
```

Comment:
pip install transformers should give priority to the local folder, but this makes it more obvious we are installing from the content of the local folder transformers.

Reply:
Should it do `pip install .[dev]` then?

Reply:
Oh yes, sorry!